packages/epo-react-lib/src/layout/SlideoutMenu/MenuGroup/MenuGroup.tsx

```diff
@@ -0,0 +1,29 @@
+import { FunctionComponent, PropsWithChildren } from "react";
+import {
+  StyledMenuGroup,
+  StyledSeparator,
+  StyledMenuGroupTitle,
+} from "./styles";
+
+interface MenuGroupProps {
+  title: string;
+}
+
+const MenuGroup: FunctionComponent<PropsWithChildren<MenuGroupProps>> = ({
```

Comment:
simple menu group that has a title, children inside a `group` role, and a separator, per ARIA